data/txHandler.go

```diff
@@ -107,6 +117,22 @@ func MakeTxHandler(txPool *pools.TransactionPool, ledger *Ledger, net network.Go
 		postVerificationQueue: make(chan *txBacklogMsg, txBacklogSize),
 		net:                   net,
 	}
+	handler.ctx, handler.ctxCancel = context.WithCancel(context.Background())
```

Comment:
this is an artifact of a previous change -- I had been passing ctx in, but then made it so I don't have to. I thought I did not contribute this line originally, so I didn't remove it. But as you point out, it is not needed and can be removed.

And yes, congestion manager should be moved to Start(). Actually, I'll probably move all this setup to start, since once the congestion manager is attached to the ERL, it's a private member, so I can only start it when I first make it (and start is a good place to put this stuff anyway)